apps/OboeTester/app/src/main/cpp/cpu/AudioWorkloadTestRunner.h

```diff
@@ -188,6 +202,8 @@ class AudioWorkloadTestRunner {
     std::atomic<bool> mIsRunning{false};
     std::atomic<bool> mIsDone{true};
     std::atomic<int32_t> mResult{0};
+    // Mutex to protect mResultText. Mutable to allow locking in const methods.
+    mutable std::mutex mResultTextMutex;
```

Comment:
If you want to lock in a getter, it is more reasonable to make the getter not const